plugins/modules/sda_fabric_devices_workflow_manager.py

```diff
@@ -4022,6 +4022,55 @@ def get_diff_merged(self, config):
 
         return self
 
+    def arrange_config_for_deletion(self, device_config):
+        """
+        Arrange the device config in such a way that the device with the role 'CONTROL_PLANE_NODE'
+        will be deleted at last.
+
+        Parameters:
+            device_config (list): Devices config details provided by the user in the playbook.
+        Returns:
+            updated_device_config (list): Updated device config details.
+        Description:
+            Loop in the device config information. If the device doesnot exists, prepend the
```

Comment:
For each device in the config, check whether it exists in the Cisco Catalyst Center.
        If the device doesn't exist, prepend it to the list. If it has a role 'CONTROL_PLANE_NODE',
        append it to ensure it is processed last. Update self.have['fabric_details'] with the new 
        order and return the updated config.